searchOne.html

```diff
@@ -30,16 +32,17 @@ <h3 class="tagline">Need to find a label for your band?</h3>
         </div>
 
         <div class="search-results">
-            <!-- ADD SEARCH RESULT MARKUP HERE -->
             <div>
                 <ul>
                     <li class="row">
                         <p>Record label 1</p>
                         <i class="fas fa-chevron-down"></i>
                     </li>
                     <li class="row">
-                        <p>Record lable 2</p>
-                        <i class="fas fa-chevron-down"></i>
+                        <a class="row" href="/searchTwo.html">
```

Comment:
@ChrysaLynn 
I added this here because this is going to be how we want to have this element be clickable. 

We _could_ technically take the `.row` off the preceding `<li>` element, but we'll leave it for now. 

But do you know _why_ I decided to wrap both the `<p>` and `<i>` elements in the `<a>`?